maintainer/travis/build_docker.sh

```diff
@@ -45,10 +45,10 @@ elif [ "$TRAVIS_OS_NAME" = "osx" ]; then
 	brew install homebrew/science/hdf5 --with-mpi --without-cxx
 
 	export TMPDIR=/tmp
-	maintainer/travis/build_cmake.sh
+	maintainer/travis/build_cmake.sh || exit 1
```

Comment:
These `exit`s are needed because the build script is no longer the last thing called in the script. Without it, all Travis builds would seemingly pass.